### PR TITLE
Switch from git:// to https://

### PR DESCRIPTION
**Pull request by leetfin**
Fix an install error, as GitHub has dropped support for insecure git:// clones.

### CODE REVIEW

**Review by TA418 — CHANGES_REQUESTED** [review]
Need to change git download link to something similar to: "git+https://github.com/bobloy/ChatterBot#egg=ChatterBot". Current means of representing versions does not exist and results in a 404 error.

**Review by delphyinium — APPROVED** [review]
This is correct. Approving.

**Comment by delphyinium** [comment]
> This is correct. Approving.

EDIT: I do not have write access. Just providing a second opinion on the fix.

**Pull request merged**
